### PR TITLE
initial slide deck implementation

### PR DESCRIPTION
**Pull request by carsonwright**

### CODE REVIEW

File: src/components/slidedeck/_slide.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+      backgroundPosition: `${this.props.slide.focus_x}% ${this.props.slide.focus_y}%`
+    }
+  }
   render() {
     return (
-      <div class="slide" style={{backgroundImage: `url(${this.props.slide.image_desktop_retina})`}}>
-        <div>
-          {this.props.slide.caption}
-        </div>
+      <div class={`${style.slide} ${this.className()}`} style={this.style()}>
```
Comment (tomprats) [inline src/components/slidedeck/_slide.js]: You can self close the tag if it's empty with <div agdhfbegs  />

File: src/components/slidedeck/index.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      overflow: hidden;
+      height: 59px;
+      line-height: 57px;
+      cursor: pointer;
+      -webkit-touch-callout: none;
+      -webkit-user-select: none;
+      -khtml-user-select: none;
+      -moz-user-select: none;
+      -ms-user-select: none;
+      user-select: none;
```
Comment (tomprats) [inline src/components/slidedeck/index.scss]: I think autoprefix adds all of the prefix versions where needed

File: src/components/slidedeck/index.js [hunk 3]
```diff
@@ -1,25 +1,120 @@
...
+            </div>
+          </div>
+          {this.loadedSlides().map((slide)=>{
+            return <Slide slide={slide} key={slide.id}/>
+          })}
+        </div>
+        <div class={style.responseContainer}>
+          <div class={style.buttons}>
+            <div class={style.me} onClick={this.answerSlide.bind(this, true)}>
+              Me
```
Comment (tomprats) [inline src/components/slidedeck/index.js]: I added some translation stuff. Very preliminary but we should definitely do something. This is fine for now

File: src/index.js [hunk 4]
```diff
@@ -76,7 +76,8 @@ if(window.TraitifyDevInitialize == true){
 
   // require("preact/devtools");   // turn this on if you want to enable React DevTools!
   // set up HMR:
-  module.hot.accept("./components/main", () => requestAnimationFrame(window.developmentLoad.reload()) );
-
+  if(typeof module != "undefined" && module.hot){
+    module.hot.accept("./components/main", () => requestAnimationFrame(window.developmentLoad.refresh()) );
```
Comment (tomprats) [inline src/index.js]: What's thisbdo?

File: src/components/slidedeck/_slide.js (hunk 1 above)
Comment (carsonwright) [inline src/components/slidedeck/_slide.js]: True

File: src/components/slidedeck/index.scss (hunk 2 above)
Comment (carsonwright) [inline src/components/slidedeck/index.scss]: Awesome I'll remove it

File: src/components/slidedeck/index.js (hunk 3 above)
Comment (carsonwright) [inline src/components/slidedeck/index.js]: I actually plan to allow people to set that text by passing in values. We could have it default though to a locale like you have. We'll get your pull request in first and then I'll try to add in the locale stuff to my content.

File: src/index.js (hunk 4 above)
Comment (carsonwright) [inline src/index.js]: It hot reloads the page so you see updates when you make changes.

**Pull request merged**